x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -288,16 +295,48 @@ const ProcessEventDotComponents = React.memo(
     const activeDescendantId = useSelector(selectors.uiActiveDescendantId);
     const selectedDescendantId = useSelector(selectors.uiSelectedDescendantId);
 
-    const logicalProcessNodeViewWidth = 360;
-    const logicalProcessNodeViewHeight = 120;
+    const isShowingEventActions = magFactorX > 0.8;
+    const isShowingDescriptionText = magFactorX >= 0.55;
+
+    /**
+     * As the resolver zooms and buttons and text change visibility, we look to keep the overall container properly vertically aligned
+     */
+    const actionalButtonsBaseTopOffset = 5;
+    let actionableButtonsTopOffset;
+    switch (true) {
+      case isShowingEventActions:
+        actionableButtonsTopOffset = actionalButtonsBaseTopOffset + 3.5 * magFactorX;
+        break;
+      case isShowingDescriptionText:
+        actionableButtonsTopOffset = actionalButtonsBaseTopOffset + magFactorX;
+        break;
+      default:
+        actionableButtonsTopOffset = actionalButtonsBaseTopOffset + 21 * magFactorX;
+        break;
+    }
+
     /**
      * The `left` and `top` values represent the 'center' point of the process node.
      * Since the view has content to the left and above the 'center' point, offset the
      * position to accomodate for that. This aligns the logical center of the process node
      * with the correct position on the map.
      */
-    const processNodeViewXOffset = -0.172413 * logicalProcessNodeViewWidth * magFactorX;
-    const processNodeViewYOffset = -0.73684 * logicalProcessNodeViewHeight * magFactorX;
+
+    const logicalProcessNodeViewWidth = 360;
```

Comment:
:red_circle: Made a comment in @kqualters-elastic PR - this is something for both of you since you're both using these: We should have them shared via an import.

Reply:
Once @kqualters-elastic PR is in, we can pull this out to a shared util